src/main/java/ui/components/pickers/MilestonePickerDialog.java

```diff
@@ -43,6 +43,15 @@ public MilestonePickerDialog(Stage stage, TurboIssue issue, List<TurboMilestone>
         state = new MilestonePickerState(originalMilestones);
         initUI();
         setupKeyEvents();
+        selectNextMilestone();
+    }
+
+    private void selectNextMilestone() {
```

Comment:
Add a header comment. I have a feeling this method name does not match what it does.